ci/build-single-ado-pr-steps.yml

```diff
@@ -10,9 +10,9 @@ steps:
   fetchDepth: 1
 
 - task: NodeTool@0
-  displayName: Use node 10
+  displayName: Use node 14
```

Comment:
>node [](http://example.com/codeflow?start=19&length=4)

instead of selecting the node version in yaml, I think we're going to need to select the node version in make.js, based on the selected buildconfig using "nvm use"

we'll need to ensure nvm (node version manager) is installed